portfolio/src/main/java/com/google/sps/servlets/BadRequestException.java

```diff
@@ -0,0 +1,7 @@
+package com.google.sps.servlets;
```

Comment:
The diff seems to be broken here - I don't know why this file appeared in the diff, I have it in base branch as well. `git diff server-testing writing-server-tests` doesn't show this file.